src/Asm89/Stack/CorsService.php

```diff
@@ -174,9 +174,16 @@ private function checkOrigin(Request $request)
             // allow all '*' flag
             return true;
         }
+
         $origin = $request->headers->get('Origin');
 
-        return in_array($origin, $this->options['allowedOrigins']);
```

Comment:
Both. The current model is just an explicit list of allowed origins. It seems to me it cannot get much simpler. :)

I do understand that this is most likely one of the more important pieces of configuration where we need to be open for extension, hence the suggestion.